test/ibc/relayer/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3"
+
+services:
+  allora:
+    image: alloranetwork/allora-chain:v0.2.14
+    volumes:
+      - ./scripts/setup_allora.sh:/allora/setup.sh
+      - $HOME/.allora/:/allora/.allorad/
+    entrypoint:
+      - /allora/setup.sh
+    platform: linux/amd64
+    environment:
+      - CHAIN_ID=allora
+      - VALIDATOR_MONIKER=validator
+      - DENOM=uallo
+      - VALIDATOR_MNEMONIC=cinnamon legend sword giant master simple visit action level ancient day rubber pigeon filter garment hockey stay water crawl omit airport venture toilet oppose
```

Comment:
maybe we pull these from an env file or something, since it's considered bad practice to have these hardcoded in the repo

Reply:
thanks for good catching